train.py

```diff
@@ -198,7 +199,37 @@
   )
 
   def make_env():
-    return nmmo.Env(config)
+    import pickle as pkl
+    import numpy as np
+    import random
+
+    import os
+    print('cwd', os.getcwd())
+    with open('./pickled_task_with_embs.pkl', 'rb') as f:
+      task_spec = pkl.load(f)
+
+    # tasks = [d[1] for d in task_spec]
+    num_tasks = len(task_spec)
+    teams = team_helper.teams
+    single_task = task_spec[0]
+
+    # make_task_fn = lambda: tasks
+    # task_spec_sampled =np.random.choice(task_spec, size=len(teams), replace=False)
+    task_spec_sampled = random.sample(task_spec, len(teams))
+    tasks = make_team_tasks(teams, task_spec_sampled)
+    make_task_fn = lambda: tasks
+    
+    # env =  nmmo.Env(config)
+    class MyNMMO(nmmo.Env):
```

Comment:
I said MyNMMO as an indicator to override the class name -- call it TestTaskNMMO or something